source/lib/api.js

```diff
@@ -28,6 +28,16 @@ export async function getApiUrl() {
 	return `${rootUrl}api/v3`;
 }
 
+export async function getRootUrl() {
+	const {rootUrl} = await optionsStorage.getAll();
+
+	if (/(^(https:\/\/)?(api\.)?github\.com)/.test(rootUrl)) {
```

Comment:
I think it's a good time for us to move from regex check here to proper URL parsing

Something like this:

```suggestion
	if ((new URL(rootUrl)).origin === 'https://api.github.com) {
```